app/Actions/Shared/ComplexStatusCheck.php

```
@@ -57,4 +57,78 @@ public function handle(Application $application)
+                    }
+                }
+            } catch (\Exception $e) {
+                // If we can't parse, treat all containers as included
+            }
+        }
+
+        $hasRunning = false;
+        $hasRestarting = false;
+        $hasUnhealthy = false;
+        $hasExited = false;
+        $relevantContainerCount = 0;
```

PHPMD (2.15.0)

Avoid excessively long variable names like $relevantContainerCount. Keep variable name length under 20. (undefined)

(LongVariable)